source/dagger/gameplay/common/aiming_system.cpp

```diff
@@ -0,0 +1,86 @@
+#define _USE_MATH_DEFINES
+
+#include "aiming_system.h"
+
+#include "core/graphics/sprite.h"
+#include "core/input/inputs.h"
+
+#include <cmath>
+#include <math.h>
+
+void AimingSystem::OnInitialize(Registry& registry_, Entity entity_)
```

Comment:
Wait, the aiming system is older than the version you put in the other branch! :D